imports/api/locations/methods.js

```diff
@@ -15,19 +15,18 @@ export const updateUserLocationAndAffordances = new ValidatedMethod({
 
     console.log("Info passed is", lat, lng)
```

Comment:
as mentioned earlier, this should be moved to the function that matches a user to a situation

Reply:
should I delete this function entirely now that I can make a request to AA from detectors/methods.js